q2_dada2/tests/data/expected/pyro-default.tsv

```diff
@@ -1,34 +1,37 @@
+362be41f31fd26be95ae43a8769b91c0	19.0	0.0	0.0	10.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
+2bf1067d2cd4f09671e3ebe5500205ca	10.0	19.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 7251b9f4e6faa33b5861aad135b5ed42	0.0	17.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 aa9dfb40c4830edc6e2118b1e6980b22	8.0	7.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 e770b18ff141c9339068443982508f7c	0.0	0.0	0.0	13.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
-2d3df7387323e2edcbbfcb6e56a02710	0.0	0.0	0.0	0.0	0.0	0.0	0.0	12.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
-5833416522de0cca717a1abf720079ac	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	12.0	0.0
+2d3df7387323e2edcbbfcb6e56a02710	0.0	0.0	0.0	0.0	0.0	0.0	0.0	12.0	0.0	0.0	6.0	0.0	0.0	0.0	0.0	0.0
+5833416522de0cca717a1abf720079ac	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	12.0	6.0
 41f67443ce8207be0c0a956c47823417	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	11.0	0.0	0.0	0.0
 e772858b8c79efc10065bc1937f37d89	0.0	0.0	0.0	10.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 d2a8811affdca18fbfc7aa973a4d8ca1	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	10.0	0.0	0.0
-8139c4ac825dae48454fb4800fb87896	0.0	0.0	0.0	0.0	0.0	0.0	8.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
+8139c4ac825dae48454fb4800fb87896	0.0	0.0	0.0	0.0	0.0	0.0	6.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 b50999f171fe6cee3c0493b50bb8926e	0.0	0.0	7.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 efff83850806ec2d1f98bdc32c9b5032	0.0	0.0	0.0	7.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 b7418eacada1319fa8b6baaf6aff06cd	0.0	0.0	6.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 385d596077ffe9ba4dcf4718d48df5a0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	5.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 4ff24fc8bbb1263940427ba778d2367b	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	5.0	0.0	0.0
 80e2a0cdbbf0ae81a93b6ba374faeb14	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	4.0
-5179f5cac20c1aa00377e4942bd7f837	0.0	0.0	3.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
-707f5ba87a63b8071f5b7cad27aa9046	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
+5179f5cac20c1aa00377e4942bd7f837	0.0	2.0	3.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
+707f5ba87a63b8071f5b7cad27aa9046	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0	2.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
 ef9ecba707ad092c6ac7bef17d0db268	0.0	0.0	0.0	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
-a7f92909575fd948fad074b72093771e	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0
\ No newline at end of file
+a7f92909575fd948fad074b72093771e	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0
+9d07a6ff18a64d9e41e70bb17e9e02c7	0.0	0.0	2.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
+857b4205ec3f91149633495d70354521	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	2.0	0.0	0.0	0.0	0.0	0.0	0.0
+4629fbb5dde5f8ee07c2259d2887a8d4	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	2.0
```

Comment:
I wasn't really expecting the pyro tests to change, but this seems reasonable. @benjjneb does this look alright? (I sorted the output to match the old one so that the diff could be checked).

Reply:
Those don't look like major shifts, but I'm also not expecting there to be any change in the pyro results. 

Is my understanding correct that only the denoise-pyro results changed, and that the denoise-single and denoise-paired tests remained identical?

Reply:
That is correct, the defaults on single and paired did not change, only pyro.

Reply:
Hmm... OK I'll run the pyro tests locally and try to identify what exactly is going on with pyro. Unexpected changes in output are no bueno, but it's probably something obvious I'm missing.

Reply:
Should be fixed in e91ee9403195c92270fd9635ee7a711a8d19ea1c

There was indeed a misindexing that had cropped up!